docs/data/toolpad/core/components/crud/crud.md

```diff
@@ -0,0 +1,356 @@
+---
+productId: toolpad-core
+title: Crud
+components: Crud, CrudProvider, List, Show, Create, Edit, CrudForm
+---
+
+# Crud
```

Comment:
Should it be CRUD or Crud?

Reply:
According to what we discussed about MUI component casing the component should be called `Crud`.